src/framework/components/app/core.clj

```diff
@@ -117,34 +117,52 @@
              :router-interceptors     router-interceptors
              :controller-interceptors controller-interceptors}))
 
+(defn mbuild-state
+  [{:keys [deps
+           http-request
+           config]}]
+  (m/>>=  (xiana/ok (create-empty-state))
+          (comp xiana/ok
+                #(assoc %
+                   :deps deps
+                   :request http-request))
+          #(if config
+             (acl-builder/init % config)
+             %)))
+
 (defn ->app
-  [{acl-cfg         :acl-cfg
+  [{_acl-cfg        :acl-cfg
     session-backend :session-backend
-    auth            :auth
+    _auth           :auth
```

Comment:
Why underscoring? Implies the localization of the binding?

Reply:
And I think they are not used. Or forgotten?

Reply:
They were not used in past versions, like they were some type of a component, but now you can pass them when you need to use, just underscoring to not forget them and to linter not complain about it.

Reply:
Just please do not remove functionality from framework. Make it possible to turn it on-off in the app. (I believe it's enough to not pass the value in the map to turn it off)